### DISCUSSION TITLE
ESP32 WROOM SX1276 PINOUT

### DISCUSSION THREAD

**Discussion by samuellbs** [jgromes/RadioLib#1087] [Q&A]
I would like to know the pinout of the sx1276 with esp32.... I use the RFM95W module....

**Comment by jgromes** [comment]
If you'd like to know the pinout maybe ask whomever you bought it from? Or ... google ...?
So far your question has nothing to do with this library.

**Reply by samuellbs** [reply]
I want to use this library, but i looked for examples and i didn´t find an example for esp32... I would like to know what connection your library needs so i can use it... (connection between esp32 and RFM95W) or can i use any esp32 GPIO ?

**Reply by jgromes** [reply]
> I want to use this library

A necessary precondition to that is knowing how to use your hardware. You don't seem to know that, since you are basically asking "how do I connect an SPI device to ESP32". You have also quite clearly not read the documentation (see https://github.com/jgromes/RadioLib/wiki/Basics#non-standard-spi-setup).
